charts/mattermost-operator/templates/mysql-operator/configmap.yaml

```
@@ -0,0 +1,22 @@
+{{- if and .Values.mysqlOperator.enabled }}
+{{- $conf := .Values.mysqlOperator.orchestrator.config }}
+{{- $_ := set $conf "RaftAdvertise" "{{ .Env.HOSTNAME }}-svc" }}
+{{- $_ := set $conf "RaftBind" "{{ .Env.HOSTNAME }}" }}
+{{- $_ := set $conf "RaftDataDir" "/var/lib/orchestrator" }}
+{{- $_ := set $conf "RaftEnabled" true }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "mysql-operator.name" . }}-orc
+  namespace: {{ .Values.mysqlOperator.namespace }}
```

Just to be inline with others:
```suggestion
  namespace: {{ template "mysql-operator.namespace" . }}
```